source/develop/release-management/features/storage/data-path-operations.html.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Data Path operations on any host
+category: feature
+authors: alitke, nsoffer, laravot, frolland
+feature_name: Data Path Operations On Any Host
+feature_modules: storage,engine,vdsm
+feature_status: WIP 4.1
+---
+
+# Data Path Operations On Any Host
+
+## Summary
+
+Until now almost all data path operations on the hosts were performed only on the elected SPM, causing a potential bottleneck.
+This feature enables data path operations on any host. 
+
+### Owner
+
+*   Adam Litke (<alitke@redhat.com>)
+*   Liron Aravot (<laravot@redhat.com>)
+*   Nir Soffer (<nsoffer@redhat.com>)
+*   Fred Rolland (<frolland@redhat.com>)
+
+### Detailed Description
+
+There are a lot of data path operations in oVirt. For example: move disk, create VM from template, create template from VM, live storage migration and more.
+These operations are performed on the SPM host, potentially creating a bottleneck in scale setups or if the operations are performed frequently.
+
+By adding a new verb 'SDM.copy_data' available on all hosts, the engine is capable to convert the data path operations flows to run on any host.
```

Comment:
I think this 'feature' is about the infrastructure that enables datapath operations on non-SPM hosts.  SDM.copy_data is just one example verb which uses the infrastructure.  We also have SDM.merge and SDM.amend_volume with many more planned.